modApiServer/src/org/aion/api/server/rpc3/types/RPCTypes.java

```diff
@@ -29,6 +28,21 @@ public Request(Integer id ,String method ,String params ,VersionType jsonRPC ){
             this.jsonRPC=jsonRPC;
         }
     }
+    /**
+    * This is the standard request body for a JSON RPC Response
+    */
+    public static final class Response {
+        public final Integer id;
+        public final Object result;
+        public final VersionType jsonRPC;
+
+        public Response(Integer id ,Object result ,VersionType jsonRPC ){
+            this.id=id;
+            this.result=result;
+            if(jsonRPC==null) throw new ParseErrorRPCException();
```

Comment:
The null check can go to the first line.

Reply:
This is based on the template and is the most concise way to define this check.

Reply:
ah, I see.